src/components/inline-inputs/inline-inputs.style.ts

```diff
@@ -49,14 +49,14 @@ const StyledContentContainer = styled.div<
 `;
 
 const StyledInlineInputs = styled.div<
-  Pick<InlineInputsProps, "gutter" | "labelWidth">
+  Pick<InlineInputsProps, "gutter" | "labelWidth" | "displayAsFlex">
```

Comment:
I realise this is something we've done elsewhere in the code but think it would be better to define interfaces in the style file and then import and extend them for the `InlineInputsProps` interface in the component file.

``` 
interface StyledInlineInputProps {
  /** Gutter prop gets passed down to Row component if false gutter value is "none" */
  gutter?: GutterOptions;
}

export interface StyledContentContainerProps extends StyledInlineInputProps {
 /** Width of the inline inputs container in percentage */
  inputWidth?: number;
}

export interface StyledInlineInputsProps extends StyledInlineInputProps {
   /** Width of a label in percentage */
  labelWidth?: number;
  /** @ignore @private */
  labelInline?: boolean;
}
```

```
export interface InlineInputsProps extends StyledContentContainerProps, StyledInlineInputsProps {
...
}
```